xml/release-notes.xml

```diff
@@ -325,6 +325,24 @@
     receive updates.
    </para>
 
+   <!-- <itemizedlist>
+    <listitem>
+     <para>
+      <package>example</package>: Replaced by
+      <package>example2</package>.
+     </para>
+    </listitem>
+   </itemizedlist> -->
+
+   <itemizedlist>
+    <listitem>
```

Comment:
Item itself looks good, but would you mind integrating it into the existing list below (i.e. below the nodejs8 item)?